engine/evd.mli

```diff
@@ -558,6 +558,9 @@ type evar_constraint = conv_pb * env * econstr * econstr
 val add_conv_pb : ?tail:bool -> evar_constraint -> evar_map -> evar_map
 val conv_pbs : evar_map -> evar_constraint list
 
+val extract_conv_pbs : evar_map ->
+      (evar_constraint -> bool) ->
+      evar_map * evar_constraint list
 val extract_changed_conv_pbs : evar_map ->
       (Evar.Set.t -> evar_constraint -> bool) ->
```

Comment:
NB the use of extract_changed_conv_pbs in evar_tactics doesn't care about the evar set so could be changed to the newly exposed extract_conv_pbs